### PR TITLE
Fixes beepsky cuffing people with no arms

### LINKED ISSUE #17709: Beepsky can cuff armless people
LEMME SHOW YOU A REAL LIMBLESS HANDCUFFS BUG RIGHT HERE

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Fixes #17709

### CODE REVIEW

**Comment by optimumtact** [comment]
I'm suprised that this is so nice

File: code/modules/mob/living/carbon/human/human_helpers.dm [hunk 1]
```diff
@@ -7,7 +7,10 @@
 	return 0
 
 /mob/living/carbon/human/canBeHandcuffed()
-	return 1
+	if(get_num_arms() >= 2)
+		return 1
```
Comment (coiax) [inline code/modules/mob/living/carbon/human/human_helpers.dm]: TRUE

File: code/modules/mob/living/carbon/human/human_helpers.dm [hunk 2]
```diff
@@ -7,7 +7,10 @@
 	return 0
 
 /mob/living/carbon/human/canBeHandcuffed()
-	return 1
+	if(get_num_arms() >= 2)
+		return 1
+	else
+		return 0
```
Comment (coiax) [inline code/modules/mob/living/carbon/human/human_helpers.dm]: FALSE

**Comment by Cheridan** [comment]
the law

File: code/modules/mob/living/carbon/human/human_helpers.dm [hunk 3]
```diff
@@ -7,7 +7,10 @@
 	return 0
 
 /mob/living/carbon/human/canBeHandcuffed()
-	return 1
+	if(get_num_arms() >= 2)
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/human_helpers.dm]: 8 legs

**Comment by optimumtact** [comment]
Kor finish this quick pls

File: code/modules/mob/living/carbon/human/human_helpers.dm (hunk 2 above)
Comment (pudl) [inline code/modules/mob/living/carbon/human/human_helpers.dm]: No way would you use true/false here. Those are used in a yes/no context, not a "number of an object" context

**Comment by KorPhaeron** [comment]
Is returning true/false actually a code standard? Everything else seems to just return 0

**Comment by coiax** [comment]
Things returning 0/1 are grandfathered in, but while you're in the area, clean it up and return TRUE and FALSE

**Pull request merged**